tests/rtl/integration/LoginPageRTL.spec.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { axe } from 'jest-axe';
+import { screen, fireEvent, waitFor } from '@testing-library/react';
+
+import App from 'components/App';
+
+import userFactory from 'fixtures/userFactory';
+import { render } from 'rtl/utils';
+
+describe('Testing Login page submit', () => {
+  const user = userFactory();
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = render(
+      <MemoryRouter initialEntries={['/login']}>
+        <App />
+      </MemoryRouter>
+    );
+  });
+
+  test('Testing the presence of fields', () => {
+    screen.getByText(/login/i);
+    screen.getByLabelText(/email/i);
+    screen.getByRole('button', { name: /submit/i });
+  });
+
+  test('Testing Login submit', async () => {
+    const emailField = screen.getByLabelText(/email/i);
+    const passwordField = screen.getByLabelText(/password/i);
+    const onSubmitLogin = screen.getByRole('button', { name: /submit/i });
+
+    fireEvent.click(onSubmitLogin);
+    expect(screen.getByText(/You must enter an email to continue/)).toBeInTheDocument();
+    expect(screen.getByText(/You must enter a password to continue/)).toBeInTheDocument();
+
+    fireEvent.change(emailField, { target: { value: user.email } });
```

Comment:
```suggestion
    userEvent.type(emailField, user.email);
```

Can we start using the recommended for Kent C. Dodds `userEvent` that also consider `keyDown`, `keyPress`, and `keyUp` events and also it looks like a little bit less boilerplate

[https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-testing-libraryuser-event](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-testing-libraryuser-event)